clc/store/store_test.go

```diff
@@ -0,0 +1,162 @@
+package store
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	badger "github.com/dgraph-io/badger/v4"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/log"
+)
+
+func bytes(s string) []byte {
+	return []byte(s)
+}
```

Comment:
I think it's better to remove this function, since `[]byte(s)` is well known and it's not much longer than `bytes(s)`.